### PR TITLE
[feature](jdbc) support call function to pass sql directly to jdbc catalog

### PR DESCRIPTION
**Pull request by morningman**
## Proposed changes

Support a new stmt in Nereids:
`CALL EXECUTE_STMT("jdbc", "stmt")`

So that we can pass the origin stmt directly to the datasource of a jdbc catalog.

show case:
```
mysql> select * from mysql_catalog.db1.tbl1;
+------+------+
| k1   | k2   |
+------+------+
|  111 | 222  |
+------+------+
1 row in set (0.63 sec)

mysql> call execute("mysql_catalog", "insert into db1.tbl1 values(1,'abc')");
Query OK, 0 rows affected (0.01 sec)

mysql> select * from mysql_catalog.db1.tbl1;
+------+------+
| k1   | k2   |
+------+------+
|  111 | 222  |
|    1 | abc  |
+------+------+
2 rows in set (0.03 sec)

mysql> call execute_stmt("mysql_catalog", "delete from db1.tbl1 where k1=111");
Query OK, 0 rows affected (0.01 sec)

mysql> select * from mysql_catalog.db1.tbl1;
+------+------+
| k1   | k2   |
+------+------+
|    1 | abc  |
+------+------+
1 row in set (0.03 sec)
```

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by morningman** [comment]
run buildall

**Comment by morningman** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/datasource/jdbc/client/JdbcClient.java [hunk 1]
```diff
@@ -467,6 +467,24 @@ protected ResultSet getColumns(DatabaseMetaData databaseMetaData, String catalog
...
+     * Execute stmt direct via jdbc
+     * @param origStmt, the raw stmt string
+     */
+    public void executeStmt(String origStmt) {
+        Connection conn = getConnection();
+        Statement stmt = null;
+        try {
+            stmt = conn.createStatement();
+            int effectedRows = stmt.executeUpdate(origStmt);
+            LOG.info("finished to execute dml stmt: {}, effected rows: {}", origStmt, effectedRows);
```
[the remark this answers is not in the document]
Comment (morningman) [reply fe/fe-core/src/main/java/org/apache/doris/datasource/jdbc/client/JdbcClient.java]: change to debug

**Comment by morningman** [comment]
run buildall

**Review by BePPPower — APPROVED** [review]
LGTM

**Review by Jibing-Li — APPROVED** [review]

**Review by AshinGau — APPROVED** [review]

**Pull request merged**